readthedocs/doc_builder/director.py

```diff
@@ -106,6 +106,21 @@ def setup_vcs(self):
         #
         # self.run_build_job("pre_checkout")
         self.checkout()
+
+        # Output the path for the config file used.
+        # This works as confirmation for us & the user about which file is used,
+        # as well as the fact that *any* config file is used.
+        if self.data.config.source_file:
+            checkout_path = self.data.project.checkout_path(self.data.version.slug)
+            command = self.vcs_environment.run(
+                "cat",
+                # Show user the relative path to the config file
+                self.data.config.source_file.replace(
+                    checkout_path + "/", ""
+                ),
```

Comment:
I don't think we need this replacement here. This should is done automatically at the API when retrieving the commands. We are storing the "full command" including the long path in the database.

```suggestion
                self.data.config.source_file,
```

* API: https://github.com/readthedocs/readthedocs.org/blob/07ef84f2be49d3377b40cb116ec0299ee965f807/readthedocs/api/v2/serializers.py#L177-L180
* Replacement function: https://github.com/readthedocs/readthedocs.org/blob/07ef84f2be49d3377b40cb116ec0299ee965f807/readthedocs/api/v2/utils.py#L265-L292

Reply:
@humitos when I tested it locally it didn't replace the path. I think this is because it isn't including the `bin` path because it's not an executable? We could expand this logic to remove that path from anywhere in the string, but I didn't want to touch too much code.